scripts/pmgr_check.py

```diff
@@ -0,0 +1,189 @@
+"""
+This script creates an atef check from a pmgr configuration.  The configuration will
+be converted into a PVConfiguration.  Note that default tolerances will be used for
+checks.
+
+An example invocation might be:
+"python scripts/pmgr_check.py cxi "KB1 DS SLIT LEF" CXI:KB1:MMS:13 test_pmgr_checkout.json"
+"""
+import argparse
+import json
+import logging
+from typing import Any, Dict
+
+import apischema
+from pmgr import pmgrAPI
+
+from atef.check import Equals
+from atef.config import ConfigurationFile, ConfigurationGroup, PVConfiguration
+
+DESCRIPTION = __doc__
+logger = logging.getLogger()
+
+
+def get_pv(prefix: str, key: str):
+    """
+    Parse key from pmgr configuration data dictionary.  Keys are of the form:
+    'FLD_ACCL' or 'FLD_BDST', denoting the suffixes to append to `prefix`.
+
+    Ignores unrecognized keys (keys without expected prefixes)
+
+    Parameters
+    ----------
+    prefix : str
+        the EPICS PV prefix
+    key : str
+        the key from a pmgr configuration data dictionary
+
+    Returns
+    -------
+    str
+        a fully qualified EPICS PV
+    """
+    if 'FLD_' in key:
+        suffix = key.removeprefix('FLD')
+    elif 'PV_' in key:
+        suffix = key.removeprefix('PV')
+    else:
+        logger.debug(f'Unrecognized key provided: {key}')
+        return
+
+    # general string fixing... ew
+    suffix_parts = suffix.split("__")
+    new_suffix_list = [":".join(substr.split('_')) for substr in suffix_parts]
+    suffix = '_'.join(new_suffix_list)
+    suffix = ".".join(suffix.rsplit(":", 1))
+    pv = prefix + suffix
+    return pv
```

Comment:
This isn't quite correct, specifically the FW_MEANS and REV_MEANS PVs are separated by colons instead of dots, e.g. currently the following PVs are checked:
```
CXI:KB1:MMS:13.FW_MEANS
CXI:KB1:MMS:13.REV_MEANS
```
But the correct PVs are:
```
CXI:KB1:MMS:13:FW_MEANS
CXI:KB1:MMS:13:REV_MEANS
```

iirc the `FLD_` PVs should always use a `.` delimiter while the `PV_` PVs should always use a `:` delimiter but I'm not 100% on that

Reply:
You are correct.  The `FLD_` PVs are the only ones that line 55 should be applied to.